wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DCMotorSim.java

```diff
@@ -252,4 +267,14 @@ public void setInputVoltage(double volts) {
     setInput(volts);
     clampInput(RobotController.getBatteryVoltage());
   }
+
+  /**
+   * Sets the input voltage for the DC motor.
+   *
+   * @param volts The input voltage.
+   */
+  public void setInputVoltage(Voltage volts) {
```

Comment:
I think this should be an overload and you just call setInputVoltage with volts.in(volts)

Reply:
It's already an overload, but I can call the original function if that's preferred.

Reply:
Yes sorry that's what I meant